README.md

```diff
@@ -171,6 +171,19 @@ interval between any two periodic sync events can be greater than the requested
 is up to the browser implemetation. Also, the interval between any two periodic sync events for a
 registration doesn't have to be the same.
 
+Since a website will most likely download resources from the network when processing a
+`periodicsync` event, multiple `periodicsync` events can cause a lot of data consumption.
+Like other service worker events, `periodicsync` is subject to the service worker execution timeout.
+Each time the browser dispatches these events, the browser will potentially be kept awake for this
+duration, which is a few minutes. This can cause battery drain.
+The browser implementation SHOULD therefore ensure an appropriate frequency of firing these events.
+The browser MAY cap the frequency of `periodicsync` events per origin and across origins. It MAY
+also enforce limits on how often it is woken up to dispatch these events. To decide what an
+appropriate frequency is, consider how often mobile OSes allow scheduling repetetive tasks. Android
+and iOS impose a max frequency of 15 and 10 minutes, respectively, at the time of writing. The
+browser implementation MAY also consider how often a user engages with a site, and adjust the
```

Comment:
I would generalise this to something like "The browser MAY also use other considerations to determine the frequency, such as user engagement with the origin."

Reply:
done.